res/keyboard/legacy_kbd_to_xml.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
```

Comment:
is this file supposed to be packaged in the installer and deployed to user's computer ?
if yes, then add it to windows NSIS uninstaller here: https://github.com/mixxxdj/mixxx/blob/master/build/nsis/Mixxx.nsi#L346
if no, the this file should be moved outside of res/ directory